### PR TITLE
Update base-nl.yaml

### PR DESCRIPTION
**Pull request by JurgenMK**

### CODE REVIEW

File: translations/base-nl.yaml [hunk 1]
```diff
@@ -30,47 +30,47 @@ steamPage:
...
-        Soon you will have to mix colors and paint your shapes with them - Combine red, green and blue color resources to produce different colors and paint shapes with it to satisfy the demand.
+        Dit spel heeft 18 opeenvolgende niveaus (Welke u bezig moeten houden voor al gauw een paar uur!) maar ik vroeg constant inhoud toe - er is veel gepland!
 
-        This game features 18 progressive levels (Which should keep you busy for hours already!) but I'm constantly adding new content - There is a lot planned!
+        Door dit spel te kopen krijgt u toegang tot de losstaande versie, welke meer mogelijkheden heeft, mede als toegang gevend aan nieuw toegevoegde onderdelen.
 
-        Purchasing the game gives you access to the standalone version which has additional features and you'll also receive access to newly developed features.
-
-        [b]Standalone Advantages[/b]
+        [b]Losstaande voordelen[/b]
```
Comment (Astavie) [inline translations/base-nl.yaml:44]: In the rest of the translation, "standalone" doesn't get translated, except here

File: translations/base-nl.yaml [hunk 2]
```diff
@@ -30,47 +30,47 @@ steamPage:
...
+            [*] Prestaties verbeteringen (Het spel draait al best goed!)
+            [*] En nog veel meer!
         [/list]
 
-        [b]This game is open source![/b]
+        [b]Dit spel is open-source![/b]
 
-        Anybody can contribute, I'm actively involved in the community and attempt to review all suggestions and take feedback into consideration where possible.
-        Be sure to check out my trello board for the full roadmap!
+        Iedereen can toevoegen, ik ben actief in de gemeenschap, en probeer alle suggesties te bekijken en terugkoppelingen te overwegen waar mogelijk.
```
Comment (Astavie) [inline translations/base-nl.yaml:72]: `can` should be `kan`

File: translations/base-nl.yaml [hunk 3]
```diff
@@ -30,47 +30,47 @@ steamPage:
     longText: >-
         [img]{STEAM_APP_IMAGE}/extras/store_page_gif.gif[/img]
 
-        shapez.io is a game about building factories to automate the creation and processing of increasingly complex shapes across an infinitely expanding map.
-        Upon delivering the requested shapes you will progress within the game and unlock upgrades to speed up your factory.
+        shapez.io is een spel over het bouwen van fabrieken, welke steeds moeilijker wordende vormen maken en verfijnen, over een oneindig grote map.
```
Comment (Astavie) [inline translations/base-nl.yaml:33]: I would translate `map` as `speelveld`

File: translations/base-nl.yaml [hunk 4]
```diff
@@ -30,47 +30,47 @@ steamPage:
...
+        Nadat u de gevraagde vormen heeft afgeleverd, komt u verder in de game, en kunt u upgrades verkrijgen om zo uw fabriek te versnellen.
 
-        As the demand for shapes increases, you will have to scale up your factory to meet the demand - Don't forget about resources though, you will have to expand across the [b]infinite map[/b]!
+        Omdat de vraag naar vormen groter wordt, moet u uw fabriek steeds groter maken om aan de vraag te voldoen - vergeet de grondstoffen niet, u zal moeten groeien over de [b]oneindige map[/b]!
+        
+        Binnenkort moet u kleuren mengen en uw vormen ermee verven - Combineer rode, groene en blauwe kleurbronnen om verschillende kleuren, en verf vormen ermee om aan de vraag te voldoen.
```
Comment (Astavie) [inline translations/base-nl.yaml:38]: "om verschillende kleuren..." this sentence isn't finished

File: translations/base-nl.yaml [hunk 5]
```diff
@@ -30,47 +30,47 @@ steamPage:
...
 
-        Soon you will have to mix colors and paint your shapes with them - Combine red, green and blue color resources to produce different colors and paint shapes with it to satisfy the demand.
+        Dit spel heeft 18 opeenvolgende niveaus (Welke u bezig moeten houden voor al gauw een paar uur!) maar ik vroeg constant inhoud toe - er is veel gepland!
```
Comment (Astavie) [inline translations/base-nl.yaml:40]: `welke` is a pretty outdated term. Could be changed to `die`.

**Comment by Astavie** [comment]
I think that was everything. There's also an overabundance of commas in the entire translation, though.

**Comment by JurgenMK** [comment]
> 
> 
> I think that was everything. There's also an overabundance of commas in the entire translation, though.

I updated the file to satisfy your comments, also removed some not really necessary commas.

**Comment by Astavie** [comment]
Looks good 👍

**Comment by tobspr** [comment]
Looks good! But it seems there are some conflicts?

**Comment by Astavie** [comment]
It seems a later PR got merged before this one, leaving this one obsolete?

**Comment by tobspr** [comment]
Closing for now

**Pull request closed without merging**